src/libsyncengine/syncpal/syncpal.h

```diff
@@ -257,8 +257,10 @@ class SYNCENGINE_EXPORT SyncPal : public std::enable_shared_from_this<SyncPal> {
         virtual void refreshTmpBlacklist();
         virtual void removeItemFromTmpBlacklist(const NodeId &nodeId, ReplicaSide side);
         virtual void removeItemFromTmpBlacklist(const SyncPath &relativePath);
-
         ExitInfo handleAccessDeniedItem(const SyncPath &relativePath, ExitCause cause = ExitCause::FileAccessError);
+        ExitInfo handleAccessDeniedItem(const SyncPath &relativePath, std::shared_ptr<Node> &remoteBlacklistedNode,
```

Comment:
We usually follow the order, `local` then `remote`.

Reply:
Done here: [c2a7256](https://github.com/Infomaniak/desktop-kDrive/pull/375/commits/c2a72567636c29932e2f6dbbdc9fddaa9c77ba36)